packages/front-end/components/Experiment/DateResults.tsx

```diff
@@ -341,20 +344,24 @@ const DateResults: FC<{
               <small className="badge badge-secondary">{resultGroup}</small>
             )}
           </h3>
-          <ExperimentDateGraph
-            yaxis="effect"
-            datapoints={datapoints}
-            label={getEffectLabel(differenceType ?? "relative")}
-            formatter={
-              differenceType === "relative"
-                ? formatPercent
-                : getExperimentMetricFormatter(metric, getFactTableById, true)
-            }
-            formatterOptions={metricFormatterOptions}
-            variationNames={variations.map((v) => v.name)}
-            statsEngine={statsEngine}
-            hasStats={!cumulative}
-          />
+          {!quantileMetricType(metric) || !cumulative ? (
```

Comment:
seems like it would be better to disable the toggle and force the non-cumulative view to render

Reply:
The toggle is for all graphs at once. So I don't want to disable it if they have one quantile metric.